File: Csharp/Laba7/Laba7/View/Output.cs
using System;

namespace View
{
	public class Output
	{
		public static void Write (string outer)
		{
			Console.Write (outer);
		}

		public static void WriteLine (string outer)
		{
			Console.WriteLine (outer);
		}

		public static void Write (int outer)
		{
			Console.Write (outer);
		}

		public static void WriteLine (int outer)
		{
			Console.WriteLine (outer);
		}

		public static void Write (double outer)
		{
			Console.Write (outer);
		}

		public static void WriteLine (double outer)
		{
			Console.WriteLine (outer);
		}

		public static void WriteMass (int rows, int columns, int[,]matrix)
		{
			string outer = "";
			for (int i = 0; i < rows; i++) 
			{
				for (int j = 0; j < columns; j++)
				{
					outer += "\t" + matrix[i,j];
				}
				outer += "\n\n";
			}
			Output.WriteLine(outer);
		}

	}
}

File: Java/Desktop/Laba7/src/by/bntu/fitr/povt/view/Output.java
package by.bntu.fitr.povt.view;

public class Output {

    private int rows,columns;
    private int matrix[][] = new int[rows][columns];
    public Output(int rows, int columns, int[][] matrix) {
        this.rows = rows;
        this.columns = columns;
        this.matrix = matrix;
    }
    public static void print(String out){
        System.out.print(out);
    }
    public static void printLine(String out){
        System.out.println(out);
    }
    public static void printMass(int rows, int columns, int[][] matrix) {
        String out = "";
        for(int i = 0;i<rows;i++){
            for (int j = 0 ;j<columns;j++){
                out += "\t" + matrix[i][j];
            }
            out += "\n\n";
        }
        Output.printLine(out);
    }
    public static void consoleClear(){
        Output.printLine("\n\n\n\n\n\n\n\n\n\n\n\n\n");
    }
}
